### PR TITLE
[stt][ACR-1782] Add description for audio streaming functions

### PR DESCRIPTION
**Pull request by stom-hwang**
### Change Description ###

Add description for audio streaming functions

### API Changes ###

 - ACR: ACR-1782

### CODE REVIEW

File: docs/application/native/guides/text-input/stt.md [hunk 1]
```diff
@@ -690,7 +709,55 @@ To set the STT options and control recording:
...
+        int ret;
+        ret = stt_stop_audio_streaming(stt);
+        if (STT_ERROR_NONE != ret)
+            /* Error handling */
+    }
+    ```
+
+  - To cancel either the recording or the audio streaming without getting the result, use the `stt_cancel()` function.
 
     The state changes to `STT_STATE_READY`.
```
Comment (omar-srbd) [inline docs/application/native/guides/text-input/stt.md:762]:
```suggestion
    The state changes to `STT_STATE_READY`:
```

**Review by omar-srbd — COMMENTED** [review]
Hello @stom-hwang,
I have reviewed the PR and left a suggestion for you.
Please incorporate the suggestion.
Thank you.

File: docs/application/native/guides/text-input/stt.md [hunk 2]
```diff
@@ -690,7 +709,55 @@ To set the STT options and control recording:
...
+        int ret;
+        ret = stt_send_audio_streaming(stt, data, size);
+        if (STT_ERROR_NONE != ret)
+            /* Error handling */
+    }
+    ```
+
+  - To stop the audio streaming and get the recognition result, use the `stt_stop_audio_streaming()` function.
+
+    The audio streaming stops and the state is changed to `STT_STATE_PROCESSING`. When the recognition result has been processed, the result is sent in the recognition result callback and the state is changed back to `STT_STATE_READY`:
```
Comment (omar-srbd) [inline docs/application/native/guides/text-input/stt.md:747]:
```suggestion
    The audio streaming stops and the state is changed to `STT_STATE_PROCESSING`. When the recognition result has been processed, the result is sent in the recognition result callback, and the state is changed back to `STT_STATE_READY`:
```

**Comment by stom-hwang** [comment]
Thank you for checking. I fixed the description that you reviewed.

**Review by omar-srbd — APPROVED** [review]
@stom-hwang, thanks for incorporating all the changes.

@asm-foysal, I am approving this PR for merging

**Pull request merged**